conditional/util/email.py

```diff
@@ -0,0 +1,51 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.utils import formatdate
+from email.mime.multipart import MIMEMultipart
+from datetime import date
+from conditional import app
+
+def send_absent_hm_attendance_emails(absent_members):
+    today_date = date.today().strftime("%m/%d")
```

Comment:
This is an ambiguous date format. Why not do `06 January` instead? This doesn't need to be terse, so be readable.

Reply:
This was done to be consistent with the emails sent out by quotefault. And at least in the subject line conciseness is better imo.

Reply:
Quotefault does not include a date in emails.￼ Please don't be terse in the body, at least.